lib/cmd/common/build.js

```diff
@@ -1,21 +1,25 @@
 module.exports = build;
 
 build.desc = "Builds a client application";
-build.usage = "\nfhc build app=<app-id> destination=<destination> version=<version> config=<config> keypass=<private-key-password> certpass=<certificate-password> download=<true|false> provisioning=<path-to-provisioning-profile> cordova_version=<cordova version>"
+build.usage = "\nfhc build app=<app-id> destination=<destination> version=<version> config=<config> keypass=<private-key-password> certpass=<certificate-password> download=<true|false> provisioning=<path-to-provisioning-profile> cordova_version=<cordova version> branch=<branch name> commit=<commit hash>"
   + "\nwhere <destination> is one of: android, iphone, ipad, ios(for universal binary), blackberry, windowsphone7, windowsphone (windows phone 8)"
   + "\nwhere <version> is specific to the destination (e.g. Android version 4.0)"
   + "\nwhere <config> is either 'debug' (default), 'distribution', or 'release'"
   + "\nwhere <provisioning> is the path to the provisioning profile"
   + "\nwhere <cordova_version> is for specifying which version of Cordova to use. Currently supported: either 2.2 or 3.3. Only valid for Android for now."
+  + "\nwhere <branch name> is the name of the git branch to build, defaults to 'master'"
+  + "\nwhere <commit hash> is the full hash of the commit to build, if not the head of the branch. Muss be used with <branch name>"
```

Comment:
Muss => Must

"Muss be used with <branch name>" - I'm not sure if I'm getting it right. Do we want here to specify both branch and hash at the same time?